### PR TITLE
Add new module file_status

### PR DESCRIPTION
**Pull request by ritze**
A simple module which shows you if a file or dir in your filesystem is available or not.

### CODE REVIEW

File: py3status/modules/file_status.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import os
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 10
+    color_available = None
+    color_unavailable = None
```
Comment (tobes) [inline py3status/modules/file_status.py]: @ultrabug I worry about having these config params.  Although the names may be appropriate for this module, there is something nice about sharing names eg color_bad as it keeps the configs consistent.

Maybe you have a nice insight on this

**Comment by ritze** [comment]
Now the module uses only the global params color_good and color_bad. Additionally the module use subprocess instead of os.system.

File: py3status/modules/file_status.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""
+Display if a file or dir exists.
+
+Configuration parameters:
+    cache_timeout: how often to run the check
+    color_available: color of format_available
+    color_unavailable: color of format_unavailable
```
Comment (tobes) [inline py3status/modules/file_status.py]: they are still here :)

**Comment by ritze** [comment]
Maybe I'm a little bit too fast with handling three branches at the same time. :-)

File: py3status/modules/file_status.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Display if a file or dir exists.
+
+Configuration parameters:
```
Comment (ultrabug) [inline py3status/modules/file_status.py]: same as the other process module, please state the defaults (and path shouldn't have one imo)

File: py3status/modules/file_status.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        if subprocess.call(["ls", self.path], stdout=fnull, stderr=fnull) == 0:
+            response['full_text'] = self.format_available
+            response['color'] = i3s_config['color_good']
+        else:
+            response['full_text'] = self.format_unavailable
+            response['color'] = i3s_config['color_bad']
+
+        return response
+
+if __name__ == "__main__":
```
Comment (ultrabug) [inline py3status/modules/file_status.py]: same comment as your process module, see github module for inspiration plz

File: py3status/modules/file_status.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+import os
+import subprocess
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 10
+    format_available = u'●'
```
Comment (ultrabug) [inline py3status/modules/file_status.py]: totally arbitrary comment, do we feel a bullet point is a good default to state if a file is present or not? I'm not sure tbh

**Comment by ritze** [comment]
Instead a bullet point we can use a [white](http://unicode-table.com/de/2690) ⚐ and [black flag](http://unicode-table.com/de/2691/) ⚑. But these symbols are not really better than the bullet point.

Or we just use text: "target exists" and "no target"

**Comment by tobes** [comment]
There have been issues with unicode chars not always being available to all users.

Personally I'd like all core modules to use ascii and allow users to then specify their own formats as preferred.

Some existing modules do however use unicode in their defaults.  Also currently python2 installs cannot use unicode in the config although there is an outstanding pull request #282 that would fix this.

What I'd really like to see is my 'themes' dream get accepted but I'm waiting for my current PRs to be accepted/rejected first.  This would have a set of themes that users can quickly enable that would alter the look/feel of py3status with minimal effort for one or all modules.

```
general {
    theme = 'SuperCool'
}
```

**Comment by ultrabug** [comment]
Same comment as the process_status and the bullet points, maybe the file path or basename would have been a better default tbh as I tend to agree with @tobes wrt unicode stuff.

Thanks a lot @ritze !

**Pull request merged**